react/features/landing/components/Landing.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import { landingIsShown } from '../actions';
+
+const links = {
+    'android': 'https://play.google.com/store/apps/details?id=org.jitsi.meet',
+    'ios': ''
+};
+
+/**
+ * React component representing mobile landing page.
+ *
+ * @class Landing
+ */
+class Landing extends Component {
+
+    /**
+     * React lifecycle method triggered after
+     * component is mount.
+     *
+     * @returns {void}
+     */
+    componentDidMount() {
+        this.props.dispatch(landingIsShown());
+    }
+
+    static propTypes = {
```

Comment:
Static members go before instance members. Another way to look at it is, comply with the existing source code.